docs/painless/painless-guide/painless-ingest.asciidoc

```diff
@@ -0,0 +1,106 @@
+[[painless-ingest]]
+=== Using Ingest Processors in Painless
+
+Some {ref}/processors.html[ingest processors] expose behavior through Painless
+methods that can be called in Painless scripts that execute in ingest pipelines.
+
+==== Method Usage
```

Comment:
```suggestion
==== Method usage
```

Reply:
We typically prefer sentence-style capitalization for headings, though there's a mix in the Painless docs.

Reply:
This mix is just legacy. Whenever the docs for Painless are modified we change the headers to the sentence style.